.github/workflows/kubectl-az.yml

```
@@ -0,0 +1,199 @@
+name: Microsoft Azure CLI kubectl plugin CI
+env:
+  GO_VERSION: 1.17 # TODO: Update
+  AKS_CLUSTER_PREFIX: kubectl-az-ci
```

Suggest using `AZURE_PREFIX` as it is used as prefix for resource group too

```suggestion
  AZURE_PREFIX: kubectl-az-ci
```